lib/rpush/daemon/apns2/delivery.rb

```diff
@@ -17,7 +17,11 @@ def initialize(app, http2_client, batch)
 
         def perform
           @batch.each_notification do |notification|
-            prepare_async_post(notification)
+            begin
+              prepare_async_post(notification)
+            rescue OpenSSL::SSL::SSLError => error
+              log_error("Notification #{notification.id} failed with SSL error")
```

Comment:
we still want to crash, right?

Reply:
I don't think that we do, actually. As I understand it, the notifications are all mixed together from different apps. My theory for the reason we're seeing the SSL error is that we are attempting to send push notifications for an inactive or disabled app. This old app doesn't have an up to date certificate. I'd like to figure out which notifications are causing this issue but I think the rest should be allowed to go through.

Reply:
Hmmm, that seems somewhat risky. It's ok with me if it's temporary.

Reply:
Risky because we'll see this error crop up more frequently? Or something else?

Reply:
Oh - this doesn't disable cert verification - apple prevents this push notification from going out anyway

Reply:
Oh ok. Yeah, I was worried about ignoring cert verification. There's still a potential attack vector here but it's probably not a big deal in practice.